src/JointAxis.cc

```diff
@@ -151,6 +154,20 @@ Errors JointAxis::Load(ElementPtr _sdf)
     this->dataPtr->dissipation = limitElement->Get<double>("dissipation",
         this->dataPtr->dissipation).first;
   }
+
+  // Load mimic joint details if provided.
+  if (_sdf->HasElement("mimic"))
+  {
+    sdf::ElementPtr mimicElement = _sdf->GetElement("mimic");
+
+    this->dataPtr->mimic = mimicJoint();
+    this->dataPtr->mimic.joint = mimicElement->Get<std::string>("joint",
+        "").first;
+    this->dataPtr->mimic.multiplier = mimicElement->Get<double>("multiplier",
+        0).first;
+    this->dataPtr->mimic.offset = mimicElement->Get<double>("offset",
+        0).first;
+  }
   else
```

Comment:
this else statement should return to its previous `if` block sibling; there are test failures due to this

Reply:
I removed the else block, as the element is optional and should not result in any errors.

Reply:
Green CI ! @